### ISSUE TITLE
Fix error in calc_offset_date for 1y

### ISSUE THREAD

**Issue by jgclark** [jgclark/NotePlan-tools#18]
When processing Birthdays, got this error:
```
Error in calc_offset_date from 2020-08-02 by 1y
```

**Comment by jgclark** [comment]
Now finding that 
```
* [x] test 2 @repeat(+1y)  @done(2020-08-02 00:15)
```
turns into
```
* [x] test 2 @repeat(+1y)  @done(2020-08-02 00:15)
* [x] test 2 @repeat(+1y)  @done(2020-08-02)
* [>] test 2 @repeat(+1y)   >2021-08-02
```
So more to fix.

**Comment by jgclark** [comment]
Fixed in v1.4.3.
